iceoryx_posh/include/iceoryx_posh/internal/popo/modern_api/base_subscriber.inl

```diff
@@ -104,46 +107,69 @@ inline cxx::expected<cxx::optional<Sample<const T>>, ChunkReceiveError> BaseSubs
     }
 }
 
-template <typename T, typename port_t>
-inline void BaseSubscriber<T, port_t>::releaseQueuedSamples() noexcept
+template <typename T, typename Subscriber, typename port_t>
+inline void BaseSubscriber<T, Subscriber, port_t>::releaseQueuedSamples() noexcept
 {
     m_port.releaseQueuedChunks();
 }
 
-template <typename T, typename port_t>
-inline void
-BaseSubscriber<T, port_t>::setConditionVariable(ConditionVariableData* const conditionVariableDataPtr) noexcept
+template <typename T, typename Subscriber, typename port_t>
+inline void BaseSubscriber<T, Subscriber, port_t>::invalidateTrigger(const uint64_t uniqueTriggerId) noexcept
```

Comment:
Why do I need to know the id to invalidate the trigger? If I provide the wrong one, nothing happens and I do not get any feedback that this happened.

Reply:
The MethodCallback requires a triggerId therefore we have to implement here a method which has the uniqueTriggerId as argument. It will be required when the subscriber has more then one event. Then you need to know which trigger you have to invalidate.